flicker/flicker.py

```diff
@@ -204,6 +204,21 @@ def _reset(self, df):
         self._nrows = None
         self._ncols = None
 
+    def _validate_column_name(self, name, names=None, dataframe_name=None):
+        if not isinstance(name, six.string_types):
+            msg = ('column name must be of type str but you provided '
+                   'type = {}').format(str(type(name)))
+            raise TypeError(msg)
+        if names is None:
+            names = self._df.columns
+        if name not in names:
```

Comment:
If `names` is None, this will raise an exception. Was an `elif` meant to be used here?